playbooks/setup-env.yaml

```diff
@@ -26,14 +26,8 @@
       args:
         chdir: '{{ zuul.project.src_dir }}'
 
-    - name: Setup environment
-      become: yes
-      command:
-        cmd: systemd-tmpfiles --create
```

Comment:
The effect is currently nil because `/run/mount` already exists on the hosts the CI is running on. Of course, that could change in the future. The additional benefit coming from the `run-host-glibc` PR is only achievable after installing the project.

Either we can duplicate the call or move the build + install steps into `setup-env.yaml`.